#! /bin/sh /usr/share/dpatch/dpatch-run
## 43_AC_USE_SYSTEM_EXTENSIONS.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: AC_USE_SYSTEM_EXTENSIONS is needed for the declaration of
## DP: `strptime'. 

@DPATCH@
diff -urNad kazehakase-0.5.6~/configure.ac kazehakase-0.5.6/configure.ac
--- kazehakase-0.5.6~/configure.ac	2009-08-24 11:59:02.000000000 +0300
+++ kazehakase-0.5.6/configure.ac	2009-08-24 11:59:37.000000000 +0300
@@ -32,6 +32,8 @@
 AM_CONDITIONAL(OS_WIN32, test "$os_win32" = "yes")
 AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
 
+AC_USE_SYSTEM_EXTENSIONS
+
 AC_ENABLE_SHARED(yes)
 AC_ENABLE_STATIC(no)
 
diff -urNad kazehakase-0.5.6~/src/net/kz-http.c kazehakase-0.5.6/src/net/kz-http.c
--- kazehakase-0.5.6~/src/net/kz-http.c	2009-08-17 17:24:43.000000000 +0300
+++ kazehakase-0.5.6/src/net/kz-http.c	2009-08-24 11:59:04.000000000 +0300
@@ -29,7 +29,5 @@
 #  include "config.h"
 #endif /* HAVE_CONFIG_H */
 
-#include <features.h>
-#define __USE_XOPEN
 #include <time.h>
 #include "gnet.h"

